src/lib/modules/charts/components/model-modal/model-modal.component.ts

```diff
@@ -102,12 +99,11 @@ export class ModelModalComponent implements OnInit {
                 this.models.forEach(projectModel => {
                     this.climateModels.forEach(model => {
                         if (projectModel.name === model.name) {
-                           model.selected = projectModel.selected;
+                            model.selected = projectModel.selected;
```

Comment:
such a nitpick but it looks like there's an extraneous space

Reply:
Ha, nah, its now properly 4 spaces which is consistent with the rest of the file. It was at three spaces.